tests/printer_test.py

```diff
@@ -482,8 +470,8 @@ def test_custom_format():
 }"""
 
     ctx = MLContext()
-    arith = Arith(ctx)
-    builtin = Builtin(ctx)
+    _ = Arith(ctx)
+    _ = Builtin(ctx)
```

Comment:
We might have to talk about this at some point I feel. Currently, these kinds of things are mainly changing the ctx, so I think this should look more like `ctx.register_dialect(Arith)`. Feel free to open a discussion on this, if you think it is important (@math-fehr maybe).

Reply:
That's a very good point, I'll add this to an issue.